src/app/components/FAQ/FAQ.tsx

```diff
@@ -1,17 +1,48 @@
+import { useEffect, useState } from "react";
+
+import { useGlobalParams } from "@/app/context/api/GlobalParamsProvider";
+import { useBtcHeight } from "@/app/context/mempool/BtcHeightProvider";
 import { getNetworkConfig } from "@/config/network.config";
+import {
+  getCurrentGlobalParamsVersion,
+  ParamsWithContext,
+} from "@/utils/globalParams";
 
-import { Section } from "./Section";
 import { questions } from "./data/questions";
+import { Section } from "./Section";
 
 interface FAQProps {}
 
 export const FAQ: React.FC<FAQProps> = () => {
+  const [paramWithCtx, setParamWithCtx] = useState<
+    ParamsWithContext | undefined
+  >();
   const { coinName } = getNetworkConfig();
+  const btcHeight = useBtcHeight();
+  const globalParams = useGlobalParams();
+
+  useEffect(() => {
```

Comment:
this part of the code is actually reused many times, lets move it to a custom hook? can do it altogether in this PR or another PR